tornettools/plot.py

```diff
@@ -269,9 +305,9 @@ def __plot_client_goodput_5MiB(args, torperf_dbs, tornet_dbs):
 
     dbs_to_plot = torperf_dbs + tornet_dbs
 
-    __plot_cdf_figure(args, dbs_to_plot, 'client_goodput_5MiB',
+    __plot_cdf_figure(args, dbs_to_plot, 'client_goodput_5MiB.{circuittype}',
```

Comment:
I think you want `f'client_goodput_5MiB.{circuittype}'` (i.e. you're missing the `f` prefix).

Reply:
Hah, done